components/mixins/experience-event/experienceevent-pushtracking.schema.json

```diff
@@ -0,0 +1,57 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/experienceevent-pushtracking",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Push notification tracking",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "push notification tracking details",
+  "definitions": {
+    "experienceevent-pushtracking": {
+      "properties": {
+        "xdm:customAction": {
```

Comment:
This is going to introduce a root level field named "customAction". Should these new fields be under a "pushNotification" parent field. Also, should this new "pushNotification" field be be added to the existing "experienceevent-marketing" mixin where we have other campaign tracking fields.

Reply:
Hi Craig, does that mean adding these keys (under a pushNotificationTracking umbrella or something) here: https://github.com/adobe/xdm/blob/master/components/datatypes/marketing.schema.json

Reply:
Yes, if they are marketing related. If it's application related then it would go under https://github.com/adobe/xdm/blob/master/components/datatypes/application.schema.json

Looking to you to know which one you think it falls under.

Reply:
Hi Craig, taking into account our conversation earlier, do you think it would make more sense to keep this mixin separate from any campaign tracking related mixins? We will use this to track the user interaction only, and keep campaign related info on a separate mixin. We intend to do exactly this for CJM since we would like to keep the mobile SDK and our push service orthogonal to it: on the device we populate this mixin with the user's interaction, and populate the CJM mixins with whatever pre-rendered data that they have provided to us.